sklearn/utils/_tags.py

```diff
@@ -184,6 +185,15 @@ class Tags:
 
     Parameters
     ----------
+    estimator_type : str or None
+        The type of the estimator. Can be one of:
+        - "classifier"
+        - "regressor"
+        - "transformer"
+        - "clusterer"
+        - "outlier_detector"
+        - "density_estimator"
```

Comment:
For a third party, are they able to define an arbitrary value here for `estimator_type`?

Reply:
this value is not validated, so it can really be anything.